streams/myth/definition.proto

```diff
@@ -14,9 +14,17 @@ message MythTokenStatusUpdate {
     saga.proto.common.myth.MythTokenState token_state = 1;
 }
 
+message WithdrawalCompletedProto {
+    string oauth_id = 1;
+    string quote_id = 2;
+    string amount = 3;
```

Comment:
amountinwei?
gasfeeinwei?

Reply:
Aah thank you !
I have changed this to `amount_in_nmyth` and `gas_fee_in_wei`